data/authors/2-kutlu.mdx

```diff
@@ -0,0 +1,10 @@
+---
+name: Sezgin Kutlu
+avatar: /static/images/avatar-kutlu.png
+occupation: TODO
+company: Artificial Intelligence Student
+email: contact@enabling-intelligence.com
+linkedin: TODO
+github: https://github.com/SeskoDk
+---
```

Comment:
@SeskoDk you need to adapt this file accordingly. this is what will be displayed on the *about* page. i already took your github images as avatar, feel free to change it though.

Reply:
Thanks for providing the configuration. 
I have already adapted my personal information.